webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.tsx

```diff
@@ -0,0 +1,65 @@
+import { confirmAlert } from 'react-confirm-alert';
+import './CustomConfirmAlert.css';
+import {
+  Button,
+  ButtonProps,
+  ButtonSize,
+  ButtonVariant,
+  Text,
+  TextVariant,
+} from '../index';
+
+type CustomConfirmAlertProps = {
+  title?: string;
+  message: string;
+  buttons: Partial<ButtonProps>[];
+};
+
+export default function CustomConfirmAlert({
+  title,
+  message,
+  buttons,
+}: CustomConfirmAlertProps): void {
+  confirmAlert({
+    customUI: ({ onClose }) => {
```

Comment:
I would add `closeOnClickOutside: true,` option to prevent from rejecting the invition to a game accidentally, but I tried and I wasn't able to make it work

Reply:
```
    closeOnClickOutside: false,
```
works for me. Will push it.